keps/sig-node/2043-pod-resource-concrete-assigments/kep.yaml

```diff
@@ -27,8 +28,7 @@ stage: stable
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-# latest-milestone: "v1.21"
-latest-milestone: "0.0"
+latest-milestone: "v1.22"
```

Comment:
You need to fix the `stable` entry in `milestone` below.

Reply:
I thought stable version - it's once and forever.